docs/source/_locale/zh_CN/LC_MESSAGES/configuration.po

```diff
@@ -401,10 +401,11 @@ msgid ""
 msgstr "确保启动 Minecraft 的 JVM 也使用 UTF-8 作为默认字符集。你可以通过以下任一操作来实现："
 
 #: ../configuration.rst:233
+#, fuzzy
```

Comment:
please review and remove all translations with the `fuzzy` mark

Reply:
is the zh_cn docs not finished yet? if it is, then the fuzzy stuffs are acceptable

Reply:
not finished yet